src/main/java/org/icgc/argo/program_service/grpc/ProgramServiceImpl.java

```diff
@@ -208,24 +208,20 @@ public void listPrograms(Empty request, StreamObserver<ListProgramsResponse> res
 
   @Override
   public void listUser(ListUserRequest request, StreamObserver<ListUserResponse> responseObserver) {
-    val shortName = request.getProgramShortName().getValue();
+    ListUserResponse response;
 
     try {
-      programService.getProgram(shortName);
-    } catch (Throwable t) {
-      responseObserver.onError(status(t));
-      return;
-    }
-
-    List<User> users;
+      val shortName = request.getProgramShortName().getValue();
+      val program = programService.getProgram(shortName);
```

Comment:
can we get invitations by programShortName directly? I tried this and it worked:
```
      val invitations = invitationService.listInvitations(shortName);
```